seip2020_practical_assignments/math/pom.xml

```diff
@@ -0,0 +1,20 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>gr.aueb.dmst.artemisdoumeni</groupId>
+    <artifactId>seip2020_practical_assignments</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>math</artifactId>
+  <name>Math operations</name>
+  <description>A project for the 3rd lab assingment</description>
+  <dependencies>
+  	<dependency>
+		<groupId>junit</groupId>
+		<artifactId>junit</artifactId>
+		<version>4.12</version>
+		<scope>test</scope>
+	</dependency>
```

Comment:
This is a system dependency and you should place it in the main POM as you did with the mockito dependency